src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryConfiguration.java

```diff
@@ -119,12 +117,20 @@ public boolean getIncludeInChangesets() {
         this.includeInChangesets = includeInChangesets;
     }
 
+    public LibraryCachingConfiguration getCachingConfiguration() {
+        return cachingConfiguration;
+    }
+
+    @DataBoundSetter public void setCachingConfiguration(LibraryCachingConfiguration cachingConfiguration) {
+        this.cachingConfiguration = cachingConfiguration;
+    }
+
     @Nonnull boolean defaultedChangelogs(@CheckForNull Boolean changelog) throws AbortException {
-      if (changelog == null) {
-        return includeInChangesets;
-      } else {
-        return changelog;
-      }
+        if (changelog == null) {
```

Comment:
Please avoid touching lines of text with only whitespace changes, so as to keep diffs minimal. You can configure your text editor to refrain from doing so.